### PR TITLE
sslscan 1.11.6

### PR DESCRIPTION
**Pull request by dsXLII**
Having fixed my local repo and incorporated changes requested by @ilovezfs , let's try this again.

### CODE REVIEW

File: Formula/sslscan.rb [hunk 1]
```diff
@@ -17,10 +17,12 @@ class Sslscan < Formula
 
   def install
     system "make"
+    mkdir_p [bin, man1]
     system "make", "install", "PREFIX=#{prefix}"
   end
 
   test do
+    assert_match version.to_s, shell_output("#{bin}/sslscan --version")
```
Comment (DomT4) [inline Formula/sslscan.rb]: This isn't really necessary given the better functionality test below :)

Comment (dsXLII) [inline Formula/sslscan.rb]: That addition was suggested by @ilovezfs . It's a quick simple test, and should be (at worst) harmless.

File: Formula/sslscan.rb [hunk 2]
```diff
@@ -17,10 +17,12 @@ class Sslscan < Formula
 
   def install
     system "make"
+    mkdir_p [bin, man1]
```
Comment (tdsmith) [inline Formula/sslscan.rb]: This seems like a regression; can you open a ticket against the sslscan repository and link to it here? Thanks!

File: Formula/sslscan.rb (hunk 1 above)
Comment (tdsmith) [inline Formula/sslscan.rb]: @ilovezfs i'll second @DomT4's idea that we don't need to recommend these; this is fine here though!

Comment (mikemcquaid) [inline Formula/sslscan.rb]: @ilovezfs Thirded 😉

Comment (ilovezfs) [inline Formula/sslscan.rb]: @mikemcquaid I must just still be traumatized by https://github.com/Homebrew/homebrew-science/pull/3538/files#diff-11c674f1c0660805e38b0c57290060f2R25

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/33ddc43b34414ea25b19c2c65c587ef64ab41814. Thank you for your contribution to Homebrew @dsXLII; we appreciate it! :smiley_cat:.

Noted the upstream fix for the directory-creation regression in https://github.com/Homebrew/homebrew-core/commit/a563cbf24fa12e26f3346ac84c8abcda41c46f8e.

**Pull request closed without merging**